src/pr-info.ts

```diff
@@ -381,8 +381,8 @@ configSuspicious["tsconfig.json"] = makeChecker(
             strictNullChecks: true,
             types: [],
             noEmit: true,
-            forceConsistentCasingInFileNames: true
-        }
+            forceConsistentCasingInFileNames: true,
+        },
```

Comment:
This one also looks strange (but not the preceding one).

Reply:
Reverted :heavy_check_mark: